src/platform/forms/containers/review/PreSubmitSection.jsx

```diff
@@ -0,0 +1,63 @@
+// libs
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+
+// platform - forms - selectors
+import { preSubmitSelector } from 'platform/forms/selectors/review';
+
+// platform - forms-system components
+import { PreSubmitSection } from 'platform/forms-system/src/js/components/PreSubmitSection';
+
+// platform - form-system actions
+import { setPreSubmit as setPreSubmitAction } from 'platform/forms-system/src/js/actions';
+
+/*
+*  RenderPreSubmitSection - Component that conditionally renders PreSubmitSection, which is default, or a custom override
+*  PreSubmitSection - Default component that renders if no CustomComponent is provided
+*  preSubmitInfo.CustomComponent - property that can be added to `preSubmitInfo` object that overwrites `PreSubmitSection`
+*/
+function PreSubmitSectionController(props) {
+  const { form, preSubmit = {}, setPreSubmit, showPreSubmitError } = props;

```

Comment:
Where does `showPresubmitError` come from?

Reply:
`form?.submission?.hasAttemptedSubmit`

will update the proptypes 😃